service/config/index.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// Configuration : Struct to contain all configuration
+type Configuration struct {
+	KeywordPriority []string `json:"keywordPriority"`
+	UserAgents []string `json:"userAgents"`
+}
+
+var config Configuration
+
+// init reads and stores data from config.json
+func init() {
+	jsonFile, err := os.Open("config.json")
+	if err != nil {
+		panic(err)
+	}
+	byteValue, ioError := ioutil.ReadAll(jsonFile)
+	if ioError != nil {
+		panic(ioError)
+	}
+	json.Unmarshal(byteValue, &config)
+	fmt.Println("[JARVIS] config.json read successfully")
+	defer jsonFile.Close()
+}
+
+// Get : It returns the configuration
+func Get() Configuration {
```

Comment:
Implement the functions as an interface. This is the correct way of making a module. Here, Config is the module as said by your package name.

func (c *Configuration) Get() *Configuration {...}